From paolo.ciarrocchi@gmail.com Mon Jul 16 14:55:11 2007
From: Paolo Ciarrocchi <paolo.ciarrocchi@gmail.com>
Date: Mon, 16 Jul 2007 23:55:05 +0200
Subject: HOWTO: Add the knwon_regression URI to the documentation
To: linux-kernel@vger.kernel.org, greg@kroah.com
Cc: michal.k.k.piotrowski@gmail.com
Message-ID: <20070716235505.51ebe812@paolo-desktop>


We should let everybody know about where the regression
list is hosted. The more is known the more it is used.

Signed-off-by: Paolo Ciarrocchi <paolo.ciarrocchi@gmail.com>
Cc: Li Yang <leoli@freescale.com>
Cc: TripleX Chung <xxx.phy@gmail.com>
Cc: Maggie Chen <chenqi@beyondsoft.com>
Cc: WANG Cong <xiyou.wangcong@gmail.com>
Cc: Tsugikazu Shibata <tshibata@ab.jp.nec.com>
Cc: IKEDA Munehiro <m-ikeda@ds.jp.nec.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/HOWTO |    3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/HOWTO
+++ b/Documentation/HOWTO
@@ -249,6 +249,9 @@ process is as follows:
     release a new -rc kernel every week.
   - Process continues until the kernel is considered "ready", the
     process should last around 6 weeks.
+  - A list of known regressions present in each -rc release is
+    tracked at the following URI:
+    http://kernelnewbies.org/known_regressions
 
 It is worth mentioning what Andrew Morton wrote on the linux-kernel
 mailing list about kernel releases:
